src/plugins/intel_cpu/tests/unit/graph/merge_transpose_reorder_test.cpp

```diff
@@ -305,4 +306,104 @@ INSTANTIATE_TEST_SUITE_P(smoke_Run_MergeTransposeReorderWithReshape,
                          ::testing::Combine(::testing::ValuesIn(input_shapes_with_reshape),
                                             ::testing::ValuesIn(test_params_with_reshape)));
 
-}  // namespace
\ No newline at end of file
+}  // namespace
+
+TEST(MergeTransposeReorderWithReshape_InplaceConflict, smoke_Run_MergeTransposeReorderWithReshape_InplaceConflict) {
```

Comment:
Since this is simply a unit test, it does make sense to give brief test suite and test names, something like that:
`TEST(MergeTransposeReorder, smoke_InplaceConflict)`

Reply:
done